Program.cs

```diff
@@ -1,4 +1,36 @@
 ﻿using System;
-Console.WriteLine ("Hello, World!");
-
-   
\ No newline at end of file
+for (; ; ) {
+   Console.Write ("Enter a password: ");
+   string password = Console.ReadLine ();
+   bool hasLength = password.Length >= 6;
+   bool hasDigit = false;
+   bool hasLower = false;
+   bool hasUpper = false;
+   bool hasSpecial = false;
+   foreach (char c in password) {
+      if (char.IsDigit (c))
+         hasDigit = true;
+      else if (char.IsLower (c))
+         hasLower = true;
+      else if (char.IsUpper (c))
+         hasUpper = true;
+      else if ("!@#$%^&*()-+".Contains (c))
+         hasSpecial = true;
+   }
+   bool var = hasLength && hasDigit && hasLower && hasUpper && hasSpecial;

```

Comment:
Don't want to save this, because you use the variable 'var' in only one. So, you can directly call it,
` if (hasLength && hasDigit && hasLower && hasUpper && hasSpecial)`